src/app/workflow/register-workflow-modal/register-workflow-modal.component.html

```diff
@@ -49,7 +49,34 @@ <h4 mat-dialog-title>Register Workflow</h4>
     </mat-step>
     <mat-step>
       <ng-template matStepLabel>Create a workflow</ng-template>
-      <div *ngIf="selectedOption?.value == 0">
+      <div *ngIf="selectedOption?.value === 0">
+        <p>
+          Navigate to GitHub to install our GitHub app on your repositories/organizations. See our
+          <a href="{{ Dockstore.DOCUMENTATION_URL }}/getting-started/github-apps.html" target="_blank" rel="noopener noreferrer">
+            documentation <mat-icon>open_in_new</mat-icon>
+          </a>
+          for how to use GitHub Apps to automatically sync your workflows with GitHub.
+        </p>
+        <div fxLayout="column" fxLayoutAlign="center center" fxLayoutGap="2rem" class="mb-2">
+          <a
+            mat-flat-button
+            type="button"
+            color="primary"
+            [href]="gitHubAppInstallationLink$ | async"
```

Comment:
Is this correct -- if I click the link the GitHub app configuration will open in a new tab, I do whatever I need to on that tab, save it and close it -- and then I come back here to the same dialog? Where I then click Finish to dismiss? If so, doesn't seem quite right.

How about:

1. Instead of being an href, clicking on the button dismisses the dialog before bringing up the GitHub app in a new tab?
2. And/or replace the Finish button with a Manage on GitHub
3. ?

Reply:
Would it make sense saying 'close' instead of finish? Not sure if I like automatically closing when the used clicks manage on GitHub but I can be convinced otherwise.

Reply:
Let's leave it as is for now -- I don't have an overwhelmingly better alternative.